src/probe_types.h

```diff
@@ -60,10 +60,10 @@ const std::vector<ProbeItem> PROBE_LIST = {
     .aliases = { "U" },
     .type = ProbeType::usdt,
     .show_in_userspace_list = true },
-  { .name = "BEGIN", .aliases = { "BEGIN" }, .type = ProbeType::special },
-  { .name = "END", .aliases = { "END" }, .type = ProbeType::special },
+  { .name = "begin", .aliases = { "begin" }, .type = ProbeType::special },
+  { .name = "end", .aliases = { "end" }, .type = ProbeType::special },
   { .name = "self", .aliases = { "self" }, .type = ProbeType::special },
-  { .name = "BENCH", .aliases = { "BENCH" }, .type = ProbeType::benchmark },
+  { .name = "bench", .aliases = { "bench" }, .type = ProbeType::benchmark },
```

Comment:
Nit: "bench" is an alias of "bench"? I think you just want to drop the alias here

Reply:
Done.